.github/actions/check_for_stc/action.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Ensure STC is installed with Pyccel'
+inputs:
+  editable:
+    description: 'editable installation'
+    default: 'false'
+runs:
+  using: "composite"
+  steps:
+    - name: check stc installation
+      shell: python
+      run: |
+        import pyccel, os
+        from os.path import exists, join
+        pyccel_loc = pyccel.__path__[0]
+        print(pyccel_loc)
+        assert(exists(join(pyccel_loc,'extensions/STC/include/stc')))
+        if "${{ inputs.editable }}" == "false":
+          assert(not exists(join(pyccel_loc,'extensions/STC/docs')))
```

Comment:
Why use strings here? Instead of:
```
if "false" == "false":
    assert ...
```
You could simply do:
```
if true:
```
You may need to change the defaults/the argument for this